specification/schemas/CarrierExceptionalStatus.json

```diff
@@ -0,0 +1,24 @@
+{
+  "type": "string",
+  "enum": [
+    "failed-delivery-attempt",
+    "failed-pre-advice",
+    "failed-collection",
+    "failed-miscellaneous",
+    "security-alert",
+    "failed-cash-on-delivery",
+    "failed-customs-clearance",
+    "require-delivery-instructions",
+    "failed-refused-delivery",
+    "failed-shipment-missing",
+    "failed-shipment-damaged",
+    "delay",
+    "relabelling",
+    "partial-collection",
+    "partial-delivery",
+    "change-in-flight"
+  ],
+  "example": "failed-delivery-attempt",
+  "default": "failed-delivery-attempt",
```

Comment:
This should have a different or no default value. Was it not called `default`? Should we add it to the enum?

Reply:
@M4tini I was about to bring this up during stand up haha